app/services/analytics_events.rb

```diff
@@ -5179,6 +5179,31 @@ def idv_warning_shown(
     )
   end
 
+  # @param [Array] error_details Full messages of the errors
```

Comment:
suggestion: Add a description that explains what this event is for

Reply:
updated in https://github.com/18F/identity-idp/pull/11615/commits/d011de2a5217dddf83ecedde0a086c8efb1f7eea